engine/src/main/java/org/hibernate/validator/constraints/SafeHtml.java

```diff
@@ -65,6 +65,13 @@
 	Tag[] additionalTagsWithAttributes() default { };
 
 	/**
+	 * @return Additional whitelist protocols which are allowed on top of the protocols specified by the
+	 * {@link #whitelistType()} for a specified tag's attribute. Adding a protocol will also whitelists the
+	 * given tag and it's attribute.
+	 */
+	Protocol[] additionalProtocols() default { };
```

Comment:
@gunnarmorling I agree. The name `additionalTagsWithAttributes` is maybe a bit off if you add `String[] protocols() default { };` to it, but that's probably preferable to adding the `@SafeHtml.Protocol`.